examples/simple/main.cc

```diff
@@ -11,19 +11,22 @@ namespace
 {
 void initTracer()
 {
-  auto exporter  = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  auto exporter = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  // Specify the Span Processor: SimpleSpanProcessor forwards spans directly to the exporter
   auto processor = std::shared_ptr<sdktrace::SpanProcessor>(
       new sdktrace::SimpleSpanProcessor(std::move(exporter)));
   auto provider = nostd::shared_ptr<trace::TracerProvider>(new sdktrace::TracerProvider(processor));
+  // Set the global trace provider
   trace::Provider::SetTracerProvider(provider);
 }
 }  // namespace
 
 int main()
 {
-  // Removing this line will leave OT initialized with the default noop
+  // Removing this line will leave OpenTelemetry SDK initialized with the default noop
```

Comment:
probably "the SDK"? - I don't know - what do you think @pyohannes?

Reply:
If we want to split some hairs, leaving out this line has the effect that no SDK is used at all. All the noop stuff is in the API.

Maybe:
> Removing this line will leave the default noop TracerProvider in place.